content/docs/csidriver/installation/helm/unity.md

```diff
@@ -47,19 +47,19 @@ Install CSI Driver for Unity using this procedure.
 
 Procedure
 
-1. Collect information from the Unity Systems like Unique ArrayId, IP address, username, and password. Make a note of the value for these parameters as they must be entered in the  `secret.yaml` and `myvalues.yaml` file.
+1. Collect information from the Unity XT Systems like Unique ArrayId, IP address, username, and password. Make a note of the value for these parameters as they must be entered in the  `secret.yaml` and `myvalues.yaml` file.
 
     **Note**: 
-      * ArrayId corresponds to the serial number of Unity array.
-      * Unity Array username must have role as Storage Administrator to be able to perform CRUD operations.
+      * ArrayId corresponds to the serial number of Unity XT array.
+      * Unity XT Array username must have role as Storage Administrator to be able to perform CRUD operations.
       * If the user is using complex K8s version like "v1.21.3-mirantis-1", use below kubeVersion check in helm/csi-unity/Chart.yaml file.
-            kubeVersion: ">= 1.21.0-0 < 1.24.0-0"
+            kubeVersion: ">= 1.22.0-0 < 1.25.0-0"
```

Comment:
This is consistent with chart.yaml . Here, we are keeping the upper limit as < 1.25.0 to accommodate any minor release. Also, Changed the reference to < 1.25.0-0 instead of <= 1.25.0-0 whereever applied for Unity.
Thanks

Reply:
Reverted the change due to https://github.com/dell/csm-docs/pull/210#discussion_r870346864